src/Core/AdminConsole/Entities/Organization.cs

```diff
@@ -249,12 +249,12 @@ public void SetTwoFactorProviders(Dictionary<TwoFactorProviderType, TwoFactorPro
     public bool TwoFactorProviderIsEnabled(TwoFactorProviderType provider)
     {
         var providers = GetTwoFactorProviders();
-        if (providers == null || !providers.ContainsKey(provider))
+        if (providers == null || !providers.TryGetValue(provider, out var twoFactorProvider))
         {
             return false;
         }
 
-        return providers[provider].Enabled && Use2fa;
+        return twoFactorProvider.Enabled && Use2fa;
     }
```

Comment:
I went for minimum changes in lines of code, to avoid looking at logical changes as well.

And I believe that guard clauses should be in top, to be consistent with the rest of the code.
It is also my personal preference